addons/point_of_sale/models/report_sale_details.py

```diff
@@ -115,7 +114,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
             cash_counted = 0
             if session.cash_register_balance_end_real:
                 cash_counted = session.cash_register_balance_end_real
-
+            cash_present = False
```

Comment:
I think you should change this var name.
It's ok for french people but for a non french speaker it'd just be a gift